### PR TITLE
Sprint 1: File mutability team

### PR DESCRIPTION
**Pull request by Antzen**
Register new ClientNameNode protocol methods to support for File Mutability. The skeleton methods are added but not yet implemented.

### CODE REVIEW

**Review by evi-liu — APPROVED** [review]

File: rice-namenode/include/ClientNamenodeProtocolImpl.h [hunk 1]
```diff
@@ -33,6 +33,10 @@ class ClientNamenodeTranslator {
   ClientNamenodeTranslator(int port, zkclient::ZkNnClient* zk_arg);
   ~ClientNamenodeTranslator();
 
+
+// Marc and pradhith and jackson
```
Comment (ghost) [inline rice-namenode/include/ClientNamenodeProtocolImpl.h]: indent

File: rice-namenode/include/ClientNamenodeProtocolImpl.h [hunk 2]
```diff
@@ -61,6 +65,8 @@ class ClientNamenodeTranslator {
   // lease manager interactions
   std::string abandonBlock(std::string input);
 
+
```
Comment (ghost) [inline rice-namenode/include/ClientNamenodeProtocolImpl.h]: unnecessary lines

File: rice-namenode/include/ClientNamenodeProtocolImpl.h [hunk 3]
```diff
@@ -33,6 +33,10 @@ class ClientNamenodeTranslator {
   ClientNamenodeTranslator(int port, zkclient::ZkNnClient* zk_arg);
   ~ClientNamenodeTranslator();
 
+
+// Marc and pradhith and jackson
+    std::string deleate(std::string);
```
Comment (ghost) [inline rice-namenode/include/ClientNamenodeProtocolImpl.h]: is this a typo?

File: rice-namenode/source/ClientNamenodeProtocolImpl.cc [hunk 4]
```diff
@@ -347,8 +362,8 @@ std::string ClientNamenodeTranslator::Serialize(
  * rpcserver.cc, which will be using a very similar - but different - function)
  */
 hadoop::common::RpcResponseHeaderProto ClientNamenodeTranslator::
-    GetErrorRPCHeader(std::string error_msg,
-                      std::string exception_classname
+GetErrorRPCHeader(std::string error_msg,
```
Comment (ghost) [inline rice-namenode/source/ClientNamenodeProtocolImpl.cc:365]: indent

**Review by ghost — DISMISSED** [review]
some nits, wait for the build to pass too

**Comment by ghost** [comment]
i think you're just failing the [linter](https://github.com/comp413-2017/RDFS/wiki/Lint) now

File: rice-namenode/source/ClientNamenodeProtocolImpl.cc [hunk 5]
```diff
@@ -86,6 +85,22 @@ ClientNamenodeTranslator::ClientNamenodeTranslator(
 
 // ----------------------- RPC HANDLERS ----------------------------
 
+
+
+
+
+std::string ClientNamenodeTranslator::append(std::string input) {
```
Comment (LINKIWI) [inline rice-namenode/source/ClientNamenodeProtocolImpl.cc:92]: Assign an owner here with `TODO`?

Ditto for the remaining handlers

**Review by LINKIWI — APPROVED** [review]
Build is clean, approving to unblock

File: rice-namenode/source/ClientNamenodeProtocolImpl.cc (hunk 5 above)
Comment (Antzen) [reply rice-namenode/source/ClientNamenodeProtocolImpl.cc:92]: Thanks for unblocking the request.

These won't be implemented until a later sprint when we implement name node functionalities, so I figured we could assign "owners" to the handlers then. 👍

**Review by ghost — APPROVED** [review]

**Pull request merged**